libparsec/crates/account/src/lib.rs

```diff
@@ -17,6 +20,8 @@ pub const AUTH_METHOD_MAC_KEY_DERIVATION_UUID: uuid::Uuid =
     uuid::uuid!("11111111-1111-1111-1111-111111111111");
 pub const AUTH_METHOD_SECRET_KEY_DERIVATION_UUID: uuid::Uuid =
     uuid::uuid!("22222222-2222-2222-2222-222222222222");
+pub const AUTH_METHOD_VAULT_KEY_DERIVATION_UUID: uuid::Uuid =
```

Comment:
`vault_key` is not derived, you should use `AUTH_METHOD_SECRET_KEY_DERIVATION_UUID` to derive `auth_method_secret_key` instead